### ISSUE TITLE
Ensembl VEP annotation is skipped with yeast genome 

### ISSUE THREAD

**Issue by amizeranschi** [nf-core/sarek#854]
Hello,

I ran the following:

```
nextflow run nf-core/sarek -r 3.1 --input yeast-test-sample-sheet.csv --outdir yeast-sarek-local-native-R64-1-1-DV-FB-ST-snpeff -profile docker --genome R64-1-1 --vep_out_format tab --skip_tools baserecalibrator --tools deepvariant,freebayes,manta,cnvkit,tiddit,vep,snpeff
```

and noticed that neither VEP, nor SNPeff, ran on my samples, using the latest Ensembl S. Cerevisiae reference genome from Igenomes.

Is this to be expected? I assumed that VEP should run fine for that genome, given that it's from Ensembl and there's definitely a VEP cache available for it.

**Comment by maxulysse** [comment]
Hi @amizeranschi your command is indeed good, but we did not actually build the containers with cache for yeast yet because until you there was no demand for it.
I do recommend using local cache, but if it's something of interest to you I can have a look into it

**Comment by amizeranschi** [comment]
@maxulysse 

I am using S. Cerevisiae for teaching to my Masters students, given that it has a small (12.6 Mbase) reference genome. I was using bcbio-nextgen until now, while currently I'm attempting to transition to nf-core. 

Using local cache for VEP is certainly a viable solution, but having things run as much out-of-the box as possible would also be very helpful for my students, in getting them to set things up for themselves on their computers.

Thanks for looking into this!

**Comment by maxulysse** [comment]
cf https://github.com/nf-core/sarek/pull/856

**Comment by amizeranschi** [comment]
@maxulysse thanks a lot for looking into this.

Could you please also add the annotations for the Bos Taurus genome UMD3.1? It's a bit older, and the latest Ensembl release that contained it was version 94.

It's a real shame that the latest Bos Taurus genome (ARS-UCD1.2) isn't in Igenomes...

**Comment by maxulysse** [comment]
Done as well, we're making a patch release for sarek soon, but dev will soon be ready to test out

**Comment by amizeranschi** [comment]
Thanks again. How can I test a dev version of sarek?

**Comment by maxulysse** [comment]
`nextflow run nf-core/sarek -r dev ...`
Probably need to do `nextflow pull nf-core/sarek` first, but wait until we merge #856

**Comment by maxulysse** [comment]
should be good to try now

**Comment by maxulysse** [comment]
@amizeranschi did it work?

**Comment by amizeranschi** [comment]
I have just set up a test job now, but it is a bit complex and will take a 3-4 hours before it finishes. I will get back to you after it's done.

```
nextflow run nf-core/sarek -r dev --input yeast-test-sample-sheet.csv --outdir yeast-sarek-local-native-R64-1-1-DV-FB-HC -profile docker --genome R64-1-1 --vep_out_format tab --joint_germline --dbsnp saccharomyces_cerevisiae.vcf.gz --tools deepvariant,freebayes,haplotypecaller,manta,cnvkit,tiddit,vep,snpeff
```

**Comment by amizeranschi** [comment]
@maxulysse Everything worked out perfectly for that test. Thanks again for looking into this.

**Comment by maxulysse** [comment]
Happy to now it, then we released sarek 3.1.1 you can use this one now
